src/structures/interfaces/TextBasedChannel.js

```diff
@@ -67,6 +68,14 @@ class TextBasedChannel {
    * @property {UserResolvable} [reply] User to reply to (prefixes the message with a mention, except in DMs)
    */
 
+  /**
+   * Options provided to control parsing of mentions by Discord
+   * @typedef {Object} MessageMentionOptions
+   * @property {MessageMentionTypes[]} [parse] Types of mentions to be parsed
```

Comment:
jsdoc typedef for `MessageMentionTypes` too?

also to keep consistent with other types should it just be `MessageMentionType`?

Reply:
Not sure on the naming convention. As a Type programmatically I agree. But its a Type for the types of mentions